### ISSUE TITLE
Can you share the hyperparameter of the code?

### ISSUE THREAD

**Issue by magicmoo** [EtoDemerzel0427/ogbn-arxiv-baselines#1]
I a student who first learn the GNN, and I have tried your code. But the accuracy is much lower than yours. Can you share the hyperparameter of your code?

**Comment by EtoDemerzel0427** [comment]
Hi @magicmoo, thank you for your feedback. However, it has been quite long since I updated this repo and I unfortunately don't remember the hyperparameter settings. I can give you some general parameter tuning suggestions if your numbers are **much** lower:
1. the default value of dropout is large, and it is used to reduce overfitting, if your models underfit, you should try a small value first.
2. don't stack too many layers of GNNs, due to the oversmoothing problem.
3. lr and epoch number should play an important role.

I'm not actively working on GNNs now, so I would suggest you find more open-source implementations (with better documentation and detailed hyperparameter settings) [here](https://ogb.stanford.edu/docs/leader_nodeprop/#ogbn-products) if you still have trouble.

**Comment by magicmoo** [comment]
thank you for your advice, I finally get the same accuracy with yours, thanks again.
